SmileIDUI/Classes/ViewModels/CaptureScreenViewModel.swift

```diff
@@ -0,0 +1,65 @@
+import Combine
+import SwiftUI
+
+@MainActor
+class CaptureScreenViewModel: ObservableObject {
```

Comment:
We should maybe prefix these with `SmileID` ? all the viewmodels so they match the class names too

Reply:
I didn't add the prefix since we're not going to be exposing view models publicly. 
They are going to be internal. I thought the SmileID prefix was meant for classes we're exposing to developers. What do you think?